lib/shoryuken/fetcher.rb

```diff
@@ -57,11 +45,24 @@ def fetch(queue, available_processors)
           @manager.async.dispatch
         end
       end
-
     end
 
     private
 
+    def receive_messages(queue, limit)
```

Comment:
Because it can be without breaking the implementation, reducing the surface area of the class.